src/main/java/ch/admin/bar/siardsuite/presenter/archive/ArchiveChooseDbmsPresenter.java

```diff
@@ -77,17 +78,16 @@ private void setListeners(MFXStepper stepper) {
         this.buttonsBox.next().setOnAction((event) -> {
             MFXRadioButton selected = (MFXRadioButton) toggleGroup.getSelectedToggle();
             if (selected != null) {
-                controller.setDatabaseType(selected.getText());
+                val selectedDbms = DbmsRegistry.findDbmsByName(selected.getText());
                 this.errorMessage.setVisible(false);
+
                 stepper.next();
-                stepper.fireEvent(new SiardEvent(UPDATE_STEPPER_DBTYPE_EVENT));
-                //fire event
+                stepper.fireEvent(new SiardEvent.DbmsSelectedEvent(SiardEvent.ARCHIVE_DBMS_SELECTED, selectedDbms));
             } else {
                 this.errorMessage.setVisible(true);
             }
         });
-        this.buttonsBox.previous().setOnAction((event) -> stage.openDialog(View.ARCHIVE_DB_DIALOG));
```

Comment:
Replacing line 89 with 90 only works for UPLOAD-Workflow. Archive-Workflow opens the `Open Siard File` Dialog instead of View.ARCHIVE_DB_DIALOG